docs/en/operations/backup.md

```diff
@@ -19,23 +19,193 @@ Keep in mind that if you backed something up and never tried to restore it, chan
+    <storage_configuration>                                                                     
+        <disks>                                                                                 
+            <backups>                                                                           
+                <type>local</type>                                                              
+                <path>/backups/</path>                                                          
+            </backups>                                                                          
+        </disks>                                                                                
+    </storage_configuration>                                                                    
+    <backups>                                                                                   
+        <allowed_disk>backups</allowed_disk>                                                    
+        <allowed_path>/backups/</allowed_path>                                                  
+    </backups>                                                                                  
+</clickhouse>
+```
+
+## BACKUP DATABASE
+
+Backup the `default` database and several of the `system` tables asynchronously:
+```sql
+BACKUP DATABASE default, TABLE system.users, \
+  TABLE system.roles, TABLE system.settings_profiles, \
+  TABLE system.row_policies, TABLE system.quotas \
+  TO Disk('backups', '1.zip') ASYNC
+```
+
+Backup the `test` database with a new name (`test2`):
+```sql
+BACKUP DATABASE test AS test2 TO Disk('backups', '1.zip')
+```
+
+Backup the database `test`:
+```sql
+BACKUP DATABASE test TO Disk('backups', '1.zip')
+
+## BACKUP TABLE
+
+Backup the table `mv_1`:
+```sql
+BACKUP TABLE mv_1 TO Disk('backups', '1.zip')
+```
+
+Backup several of the `system` tables:
+```sql
+BACKUP TABLE system.users, TABLE system.roles, \
+  TABLE system.settings_profiles, TABLE system.row_policies, \
+  TABLE system.quotas TO Disk('backups', '1.zip')
+```
+
+Backup with a specific username:
+```sql
+BACKUP TABLE system.users, TABLE system.roles \
+  TO Disk('backups', '1.zip'), user="u2"
+```
+
+Backup a table with a new name:
+```sql
+BACKUP TABLE test.table AS test.table2 TO Disk('backups', '1.zip')
+```
+
+Backup a table asynchronously:
+```sql
+BACKUP TABLE test.table TO Disk('backups', '1.zip') ASYNC
+```
+
+### Compress and password protect a backup
+
+Use `lzma` compression, and password protect the backup:
+```sql
+BACKUP TABLE test.table TO Disk('backups', '1.zip') \
+  SETTINGS compression_method='lzma', compression_level=3, password='qwerty'
+```
+
+Add an `id` to the backup (the id is used during `RESTORE`):
+```sql
+BACKUP TABLE test.table TO Disk('backups', '1.zip') \
+  SETTINGS id='first' ASYNC
+```
+
+### Incremental table backup
+
+Incrementally backup a table:
+```sql
+BACKUP TABLE test.table2 TO {incremental_backup_name} \
```

Comment:
I haven't tried this myself, but it looks like incremental_backup_name needs to another Disk(...)